1208-projects/user_edits_project.feature

```diff
@@ -0,0 +1,106 @@
+Feature: User edits Project
+
+    @xxx
+    Scenario: User opens the Project Wizard
+        Given User clicks the Create New Project Button
+        When they see a empty Project Wizard 
+        Then they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        And they see a Save Button
+        And they see a Next Button
+
+    @xxx
+    Scenario: User opens a Project in the Project Wizard
```

Comment:
Denn virker som en kopi av den over - en av de bør fjernes

Reply:
Kanskje vi kan forenkle disse to ved å lage et scenario for å se Project Wizard?
```
    Scenario: User sees the Project Wizard
        Given User with access rights for Project
        When they open the Project Wizard
        Then they see tabs:
            | Description  |
            | Participants |
            | Financing    |
        And they see the Description tab
        And they see fields:
            | Title                    |
            | Summary                  |
            | Coordinating Institution |
            | Start Date               |
            | End Date                 |
            | Internal Project Code    |
        And they see a Support Button
        And they see a Save Button
        And they see a Next Button
```

Så kan vi erstatte det med
```
   Then they see the Project Wizard
```

I de to eksisterende scenarioene

Reply:
Eg har endret omtrent tilsvarende som Eirik foreslår